README.md

```diff
@@ -1,7 +1,7 @@
 gRPC - An RPC library and framework
 ===================================
 
-gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can run anywhere. It enables client and server applications to communicate transparently, and makes it easier to build connected systems.
+gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can be used and run anywhere and anytime. It not only enables client and server applications to communicate transparently, but also makes it easier to build connected systems.
```

Comment:
Replace "be used and run..." with "be used in variety of [languages and platforms](https://grpc.io/about/#osp)."